test/Influencers/InfluencersTest.php

```diff
@@ -459,7 +454,39 @@ public function testTagList() {
          'influencers' => array($this->infUid, $this->infUid2),
          'tags' => array($this->infTag, $this->infTag.'inf2')));
 
-   } // End function testTagList()
+   }
+
+   public function testChannelsAdd() {
+      $ownership = 'reference';
+
+      $result = Traackr\Influencers::channelsAdd([
+          'influencer' => $this->infUid,
+          'url' => $this->channel,
+          'ownership' => $ownership
+      ]);
+
+      $this->assertNotEmpty($result['uid']);
+      $this->assertEquals($this->infUid, $result['influencer_uid']);
+      $this->assertEquals('BLOG', $result['platform']);
+      $this->assertNotEmpty($result['ownership']);
```

Comment:
Asserting "not empty" for these fields in case someone changes the underlying data for the channel.